### DISCUSSION TITLE
Can I expose nextcloud-aio-database port to host?

### DISCUSSION THREAD

**Discussion by joezimjs** [nextcloud/all-in-one#3006] [Questions] [answered]
I'm barely getting by with my Docker knowledge, so I'm not really sure how to set it up or if it's possible with the way things are set up, but is there a way I can open up the nextcloud-aio-database's port to the host? I have a pgAdmin desktop client and I'd rather not set up the Docker version too if I can help it, so I'd like that desktop client to be able to connect to the Postgres DB in that container.

**Comment by szaimen** [comment, marked as answer]
> Can I expose nextcloud-aio-database port to host?

No, you cannot

**Comment by joezimjs** [comment]
OK, so I need to use the pgadmin Docker setup, right? I don't really understand how to do what the Readme says:

> For the container to connect to the `aio-database`, you need to connect the container to the docker network `nextcloud-aio` and use `nextcloud-aio-database` as database host, `oc_nextcloud` as database username and the password that you get when running `sudo docker exec nextcloud-aio-nextcloud grep dbpassword config/config.php` as the password.

How do I "connect the container to the docker network `nextcloud-aio` and use `nextcloud-aio-database` as database host"?

**Reply by szaimen** [reply]
Why are you trying to install pgadmin in the first place if I may ask?

**Reply by joezimjs** [reply]
See this issue https://github.com/nextcloud/news/issues/2293
